mycore-base/src/main/java/org/mycore/common/xml/MCRURIResolver.java

```diff
@@ -1826,4 +1829,44 @@ public Source resolve(String href, String base) throws TransformerException {
         }
     }
 
+    /**
+     * Resolves arbitrary static methods of arbitrary classes. Parameters are considerd to be of type
+     * {@link java.lang.String}.
+     * <br/><br/>
+     * <strong>Invocation</strong>
+     * <pre><code>function:&lt;class name&gt;:&lt;method name&gt;:&lt;param1&gt;:&lt;param2&gt;</code></pre>
+     * <br/>
+     * <strong>Example</strong>
+     * <pre><code>function:de.uni_jena.thunibib.user.ThUniBibUtils:getLeadId:id_connection:foobar;</code></pre>
+     *
+     * @author shermann (Silvio Hermann)
+     * */
+    private static class MCRFunctionResolver implements URIResolver {
```

Comment:
Should be a separate class file and included with property `MCR.URIResolver.ModuleResolver.callJava=org.mycore.`...`.MCRFunctionResolver`